ldk/javascript/examples/self-test-loop/src/tests/filesystem/index.ts

```diff
@@ -402,7 +402,17 @@ export const testFileStat = (): Promise<boolean> =>
         writeOperation: WriteOperation.overwrite,
         writeMode: 0o755,
       });
+
       const fileInfo = await filesystem.stat(filePath);
+
+      console.log('type of modTime: ', typeof fileInfo.modTime);
+      console.log('modtime value: ', fileInfo.modTime);
+      console.log('DateofmodTime', new Date(fileInfo.modTime));
+
+      console.log('keys:', Object.keys(fileInfo.modTime));
+
+      console.log('modTime Passed into newDate', new Date('2021-09-20 23:04:07.237590597'));
```

Comment:
Should these console logs be removed before submitting?

Reply:
@MichaelOliveMcCarrick Yes, I'm still working on polishing this PR lol.